queries/c/textobjects.scm

```diff
@@ -54,8 +54,10 @@
 (preproc_else
   (_) @statement.outer)
 
-(parameter_list
-  (parameter_declaration) @parameter.inner)
+((parameter_list
+  (parameter_declaration) @parameter.inner . ","? @_end)
+ (make-range! "parameter.outer" @parameter.inner @_end))
```

Comment:
We should use the `#` syntax for predicates

```suggestion
 (#make-range! "parameter.outer" @parameter.inner @_end))
```

Reply:
@stsewd I guess if we'd had a ternary `make-range!` with optional prefix and suffix we can remove all the `.start` code because we can express then everything with queries. We should be able to implement your feature with the find_best_match function.